packages/scm/src/browser/scm-resource-command-registry.ts

```diff
@@ -0,0 +1,64 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { injectable, named, inject } from 'inversify';
+import { ContributionProvider } from '@theia/core';
+
+export const ScmResourceCommandContribution = Symbol('ScmResourceCommandContribution');
+
+export interface ScmResourceCommandContribution {
```

Comment:
Is it used by plugin system? I've expected that Theia and vscode git extension should use the same API. I don't think we need 2 different, especially if we want to get rid of Theia extension this year.